anycloud/docs/how-to/aws-postgres.md

```diff
@@ -0,0 +1,132 @@
+
+1) First download Prisma's example and install dependencies
+
+```
+curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=2 prisma-examples-latest/typescript/rest-express
+mv rest-express anycloud-node-aws-psql
+```
+
+Install npm dependencies:
+
+```
+cd anycloud-node-aws-psql
+npm install
+```
+
+2) Add `npm scripts` to the `package.json` file
+
+```
+  "scripts": {
+    "build": "tsc",
+    "clean": "rm -rf dist",
+    "start": "node ./dist/src/index.js",
+  }
+```
+
+3) Define the `Dockerfile`: 
+
+```bash
+FROM node:lts
+
+COPY . .
+
+RUN npm install
+RUN npm run build
+CMD npm run start
+```
+
+4) Initialize a `git` repository
+
+```bash
+git init
+git add -A
+git commit -m "Initial commit"
+```
+
+5) Use the AnyCloud CLI to create an `anycloud.json` file in the project directory and define a `Deploy Config`. You will need to pick a name or alias for the `Deploy Config`. The initial value will be `staging`. Since there is only `Credentials`, the CLI will default to that for your new `Deploy Config`.
+
+```bash
+$ anycloud config new
+? Name for new Deploy Config › staging
+? Pick Credentials to use ›
+❯ mystartup-aws
+  Create new Credentials
+? Region name › us-east-1
+? Virtual Machine Type › m5.large
+? Do you want to add another region to this Deploy Config? › n
+Successfully created "staging" Credentials
+```
+
+6) Deploy the Node.js server to your AWS account using the AnyCloud CLI. The only argument required is the value of one of the keys in `anycloud.json` to reference a deploy configuration. We use `staging` which we previously defined:
+
+```bash
+$ anycloud new
+? Pick Deploy Config for App ›
+❯ staging
+? Optional App name ›
+▇ Creating new App
+```
+
+7) Now go to [AWS RDS](https://console.aws.amazon.com/rds/home) and select the PostgreSQL instance in order to include the [Security Groups](https://console.aws.amazon.com/ec2/v2/home?#SecurityGroups) for the newly created AnyCloud app. It will have the format `anycloud-${app-id}-${uuid}`. First click `Modify`
+
+<img src="../assets/modify-rds.png" />
+
+Now scroll down to the `Connectivity` section and add an additional security group from the dropdown menu
+
+<img src="../assets/add-connectivity.png" />
+
+8) Now point the Node.js REST Prisma ORM to the PostgreSQL instance you just created by setting the [datasource](https://www.prisma.io/docs/concepts/database-connectors/postgresql/) in the Prisma schema file at `rest-express/prisma/schema.prisma`
+
+```
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+And define a `.env` file as follows
+
+```
+DATABASE_URL=postgresql://USER:PASSWORD@HOST:PORT/DATABASE
+```
+
+9) Finally, let's make sure we run migrations when building our Node.js app by adding a `npm build` script to `package.json`
+
+```
+npx prisma migrate deploy
+```
+
+10) Now run `anycloud upgrade -e .env` to upgrade the new app and have the changes take effect.
```

Comment:
What happen if we do these last 3 steps before deploying the first time and after its deployed just update the postgres security group? that would not work?